tests/test_registration_widget.py

```diff
@@ -65,3 +65,70 @@ def test_sample_dropdown_index_changed_with_valid_index(
         registration_widget._moving_image.name
         == registration_widget._sample_images[1]
     )
+
+
+def test_scale_moving_image_no_atlas(
+    make_napari_viewer_with_images, registration_widget, mocker
+):
+    mocked_show_error = mocker.patch(
+        "brainglobe_registration.registration_widget.show_error"
+    )
+    registration_widget._atlas = None
+    registration_widget.adjust_moving_image_widget.scale_image_signal.emit(
+        10, 10
+    )
+    mocked_show_error.assert_called_once_with(
+        "Sample image or atlas not selected. "
+        "Please select a sample image and atlas before scaling"
+    )
+
+
+def test_scale_moving_image_no_sample_image(
+    make_napari_viewer_with_images, registration_widget, mocker
+):
+    mocked_show_error = mocker.patch(
+        "brainglobe_registration.registration_widget.show_error"
+    )
+    registration_widget._moving_image = None
+    registration_widget.adjust_moving_image_widget.scale_image_signal.emit(
+        10, 10
+    )
+    mocked_show_error.assert_called_once_with(
+        "Sample image or atlas not selected. "
+        "Please select a sample image and atlas before scaling"
+    )
+
+
+@pytest.mark.parametrize(
+    "x_scale_factor, y_scale_factor",
+    [
+        (0.5, 0.5),
+        (1.0, 1.0),
+        (2.0, 2.0),
+        (0.5, 1.0),
+        (1.0, 0.5),
+    ],
+)
+def test_scale_moving_image(
+    make_napari_viewer_with_images,
+    registration_widget,
+    mocker,
+    x_scale_factor,
+    y_scale_factor,
+):
+    mock_atlas = mocker.patch(
+        "brainglobe_registration.registration_widget.BrainGlobeAtlas"
+    )
+    mock_atlas.resolution = [20, 20, 20]
+    registration_widget._atlas = mock_atlas
+
+    curr_size = registration_widget._moving_image.data.shape
```

Comment:
```suggestion
    current_size = registration_widget._moving_image.data.shape
```
(+renaming where re-used)?
Extremely nitpicky (sorry!), but all other variable names are not abbreviated.